### DISCUSSION TITLE
How to sign in as existing user in different homeserver?

### DISCUSSION THREAD

**Discussion by abhikjain360** [syphon-org/syphon#371] [Q&A] [answered]
I have an account in a homeserver that is not `matrix.org`, but this homeserver has new registrations disabled. I don't see an option to change the homeserver when logging in as an exisiting user. Is there anyway to do so?

I have tried desktop (Linux) client an mobile (Android) client, and doesn't seems to be an option in either.

Also as a side note, when trying to register as a new user using Android client on an homeserver that doesn't allow registration, the android client after one try gray's out the continue button but doesn't tell why we can not continue. The desktop client shows that registration has been disabled. Should I open an issue for it?

**Comment by 0x1a8510f2** [comment, marked as answer]
You have two options when it comes to signing into a different homeserver than matrix.org:

1) Use your full MXID ie., `@alice:alice.net` instead of just the local part (`alice`). This will sign you into `alice.net`.

2) Click the magnifying glass icon in the username field. This will bring up a home server search list. If your home server is in that list, select it. If not, type its address into the search bar (ie., `alice.net`) and select it once done typing. This will sign you into that homeserver even if you just use the localpart.

Also, yes, by all means open an issue as this sounds like a bug or design flaw.

Hope that helps!
